perfectscale/README.md

```diff
@@ -0,0 +1,66 @@
+# Agent Check: Perfectscale
+
+## Overview
+
+
+PerfectScale is a comprehensive cloud-agnostic solution that enables teams with continuous and autonomous Kubernetes optimization capabilities. Our main goal is to empower teams to effortlessly manage their infrastructure, make quick, data-driven decisions, and reach peak performance at the lowest possible cost.
+
+This integration enables you to receive PerfectScale Alerts directly into your Datadog dashboard, ensuring you stay informed about any resilience risks identified in your Kubernetes environment, and helping you prioritize tasks to eliminate the issues before they impact performance and user experience.
```

Comment:
Can you clarify what data your integration specifically monitors and what type of data is specifically sent into Datadog?

Reply:
Done